<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>test1</class>
 <widget class="QMainWindow" name="test1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>test1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label0">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 170, 255);
font: 20pt &quot;方正胖娃_GBK&quot;;
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>100</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 170, 255);
font: 20pt &quot;方正胖娃_GBK&quot;;
background-color: rgb(255, 255, 255);

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>100</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 170, 255);
font: 20pt &quot;方正胖娃_GBK&quot;;
background-color: rgb(0, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 170, 255);
font: 20pt &quot;方正胖娃_GBK&quot;;
background-color: rgb(255, 0, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>200</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 170, 255);
font: 20pt &quot;方正胖娃_GBK&quot;;
background-color: rgb(255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label5">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>200</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 170, 255);
font: 20pt &quot;方正胖娃_GBK&quot;;
background-color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label6">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>300</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 170, 255);
font: 20pt &quot;方正胖娃_GBK&quot;;
background-color: rgb(0, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label7">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>300</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 170, 255);
font: 20pt &quot;方正胖娃_GBK&quot;;
background-color: rgb(0,0, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label8">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>300</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 170, 255);
font: 20pt &quot;方正胖娃_GBK&quot;;
background-color: rgb(255, 170, 127);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 170, 255);
font: 15pt &quot;方正胖娃_GBK&quot;;
background-color: rgb(0, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label8_0">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>440</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 9pt &quot;方正康体简体&quot;;
background-color: rgb(255, 170, 127);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label8_1">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>440</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 9pt &quot;方正康体简体&quot;;
background-color: rgb(255, 170, 127);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label8_3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>465</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 9pt &quot;方正康体简体&quot;;
background-color: rgb(255, 170, 127);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label8_4">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>465</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 9pt &quot;方正康体简体&quot;;
background-color: rgb(255, 170, 127);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label8_6">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>490</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 9pt &quot;方正康体简体&quot;;
background-color: rgb(255, 170, 127);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label8_7">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>490</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 9pt &quot;方正康体简体&quot;;
background-color: rgb(255, 170, 127);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label8_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>440</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 9pt &quot;方正康体简体&quot;;
background-color: rgb(255, 170, 127);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label8_5">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>465</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 9pt &quot;方正康体简体&quot;;
background-color: rgb(255, 170, 127);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label8_8">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>490</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 9pt &quot;方正康体简体&quot;;
background-color: rgb(255, 170, 127);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label9_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>440</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 9pt &quot;方正康体简体&quot;;
background-color: rgb(255, 170, 127);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label9_0">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>440</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 9pt &quot;方正康体简体&quot;;
background-color: rgb(255, 170, 127);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label9_8">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>490</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 9pt &quot;方正康体简体&quot;;
background-color: rgb(255, 170, 127);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label9_7">
    <property name="geometry">
     <rect>
      <x>435</x>
      <y>490</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 9pt &quot;方正康体简体&quot;;
background-color: rgb(255, 170, 127);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label9_1">
    <property name="geometry">
     <rect>
      <x>435</x>
      <y>440</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 9pt &quot;方正康体简体&quot;;
background-color: rgb(255, 170, 127);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label9_4">
    <property name="geometry">
     <rect>
      <x>435</x>
      <y>465</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 9pt &quot;方正康体简体&quot;;
background-color: rgb(255, 170, 127);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label9_5">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>465</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 9pt &quot;方正康体简体&quot;;
background-color: rgb(255, 170, 127);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label9_3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>465</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 9pt &quot;方正康体简体&quot;;
background-color: rgb(255, 170, 127);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label9_6">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>490</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 9pt &quot;方正康体简体&quot;;
background-color: rgb(255, 170, 127);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>470</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 170, 255);
font: 12pt &quot;方正胖娃_GBK&quot;;
</string>
    </property>
    <property name="text">
     <string>初始状态</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>470</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 170, 255);
font: 12pt &quot;方正胖娃_GBK&quot;;
</string>
    </property>
    <property name="text">
     <string>目标状态</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>470</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 170, 255);
font: 12pt &quot;方正胖娃_GBK&quot;;
</string>
    </property>
    <property name="text">
     <string>------&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>701</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="geometry">
     <rect>
      <x>363</x>
      <y>116</y>
      <width>137</width>
      <height>54</height>
     </rect>
    </property>
    <property name="title">
     <string>八数码问题：</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
